mutant.yml

```diff
@@ -4,4 +4,5 @@ includes:
 integration: rspec
 requires:
 - mutant
+- mutant/integration/rspec
```

Comment:
@mbj Do we need to do the same thing for the minitest integration?

Reply:
@dgollahon I'm not considering this the final solution. I do not want to load rspec and minitest together at this point.

For now I chose this route as mutant is specified via rspec, for minitest I want to use a different solution, likely a separate mutant config altogether.

Reply:
Cool, makes sense--just wanted to make sure that possible issue wasn't overlooked.